addon/ng2/tasks/npm-install.js

```diff
@@ -0,0 +1,17 @@
+/* jshint node: true, esversion: 6 */
```

Comment:
I also don't like that part: https://github.com/ember-cli/ember-cli/blob/master/lib/tasks/npm-task.js#L18. It causes problems on Windows, there were a lot of people complaining about it, the fix was to `npm link npm`.